src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -50,9 +50,9 @@
         public static final double WHEELBASE = 1.0;
 
         public static final DrivetrainFeedforwardConstants FEEDFORWARD_CONSTANTS = new DrivetrainFeedforwardConstants(
-                0.072746,
-                0.0032181,
-                0.30764);
+                2.33,
+                0.0766,
+                0.236);
```

Comment:
Are these the correct units? I haven't been able to find the WPILib auto using these anywhere, but I have found 2910's using them, which implies that they should be in inches, not meters.

Reply:
UH h i straight up do not know lol

Reply:
Judging by the original values (which we know are somewhat close since they work on the swerve drivebase), these units seem off. Looking at the characterization data, the inches values seem closer to what we had before, so we should probably use those.

Reply:
👍